chromadb/segment/impl/manager/local.py

```diff
@@ -97,7 +106,9 @@ def reset_state(self) -> None:
             instance.stop()
             instance.reset_state()
         self._instances = {}
-        self._segment_cache = defaultdict(dict)
+        self.segment_cache[SegmentScope.METADATA].reset()
```

Comment:
we don't need to to cache metadata segments with LRU, since its all one segment in sqlite